File: src/examples/node/webhooks/update-webhook.js
const Easypost = require('@easypost/api');
const dotenv = require('dotenv');

dotenv.config({ path: '/Users/jhammond/git/easypost/easypost-tools/.env' });
const api = new Easypost(process.env.EASYPOST_TEST_API_KEY);

api.Webhook.retrieve('hook_123...').then(webhook => {
  webhook.save().then(console.log).catch(console.log);
}).catch(console.log);

File: src/examples/python/webhooks/update_webhook.py
import os

import easypost
from dotenv import load_dotenv

load_dotenv()
easypost.api_key = os.getenv('EASYPOST_TEST_API_KEY')

webhook = easypost.Webhook.retrieve("hook_123...")

webhook.update(
    webhook_secret="sécret",
)

print(webhook)
